node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -112,6 +112,11 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
          * to jump past the top navigation
          */
         var setUpSkipLinks = function() {
+            // Index page has a unique structure.
+            if ($('#index-page-container').length > 0) {
```

Comment:
My preference would still be to think of a name different than `oae-main-content` and use that as the id for the main content container in both the lhnav widget and index page. I think that's cleaner than introducing an implicit dependency between several widgets.